### DISCUSSION TITLE
Help:Avaloniaui TextBlock can not display completely

### DISCUSSION THREAD

**Discussion by NeverMorewd** [MrJul/Estragonia#22] [Q&A]
Attachment is my code
[hellogodot.zip](https://github.com/user-attachments/files/17484716/hellogodot.zip)

**Comment by NeverMorewd** [comment]
![b26f9a4fb5fd15f1ba97aa2ecb0bb64](https://github.com/user-attachments/assets/fcc17a0f-0213-4551-8168-01bcb73e74cb)

**Comment by MrJul** [comment]
Your control in Godot is only 40x40 pixels, so that's the size that will be used to render the Avalonia content:
![image](https://github.com/user-attachments/assets/cf9c5ce7-c007-47b2-a8a7-47c515c3de97)

You need to give it more space.
